module/VuFind/src/VuFind/ILS/Driver/Evergreen.php

```diff
@@ -814,4 +881,22 @@ public function findReserves($course, $inst, $dept)
         // TODO
         return [];
     }
+
+    /**
+     * Format date
+     *
+     * This formats a date coming from Evergreen for display
+     *
+     * @param string $date The date string to format; may be null
+     *
+     * @throws ILSException
+     * @return string The formatted date
+     */
+    private function _formatDate($date)
```

Comment:
As a matter of style, VuFind generally favors protected over private to ensure that everything is overrideable in custom subclasses. You might want to adjust this method accordingly.

Reply:
Thanks for the feedback; I've made that method protected rather than private.

Reply:
Thanks!